addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -34,23 +34,25 @@ const StorePatch = {
         this.starred = Record.one("Thread");
         this.history = Record.one("Thread");
     },
+    async initialize() {
+        await Promise.all([
+            this.fetchStoreData("failures"),
+            this.fetchStoreData("systray_get_activities"),
```

Comment:
Note: this was wrongly hard-coded in python, now back to being conditional (to this bundle in particular)